src/command.rs

```diff
@@ -13,13 +13,15 @@ pub fn find_selected_version(command: &str) -> Option<String> {
 }
 
 pub fn find_system_bin(command: &str) -> Option<PathBuf> {
-    let shim_dir = config::shim_dir();
-    let path = env::var("PATH").expect("env var PATH is not defined");
-    let paths: Vec<_> = env::split_paths(&path)
-        .filter(|p| p != &shim_dir)
-        .collect();
+    let system_path = env::var("PATH").unwrap();
```

Comment:
I don't think that we should panic if the `PATH` is not set. We're trying to find a system bin from a command name. If there's no path then we can't find it that's it. We should return `None`